sw/common/demo_system.h

```diff
@@ -13,6 +13,9 @@
 #define UART0_BASE 0x80001000
 #define DEFAULT_UART UART_FROM_BASE_ADDR(UART0_BASE)
 
+#define SPI0_BASE 0x80003000
```

Comment:
I've chosen the same address for the PWM. I guess we can wait until the first PR is merged and then the second one has to change the value to `0x80004000` or so